src/main/java/com/brandon3055/draconicevolution/capabilities/IPlayerCapabilities.java

```diff
@@ -0,0 +1,27 @@
+package com.brandon3055.draconicevolution.capabilities;
+
+/**
+ * Created by FoxMcloud5655 on 26/08/2019.
+ */
+public interface IPlayerCapabilities {
```

Comment:
the name "Player Capabilities" is incredibly vague/generic and potentially a little confusing. Think about it Your adding a capability called player capabilities to the player's capabilities.  And don't forget there's also the other player capabilities (player.capabilities) 

Consider Forge Energy for example. The FE interface is called IEnergyStorage not ITileEntityCapabilities. That just would not make sense.

Consider something like the following 
IPlayerCapabilities -> IShieldState
PlayerCapabilities -> ShieldState
PlayerCapabilitiesProvider -> ShieldStateProvider
PlayerCapabilitiesStorage -> ShieldStateStorage

Reply:
Never made player capabilities before, so thanks.  I named it this way because I thought it would be future-proof in case you wanted to add more later.  I'll change it.